javagen/src/main/java/com/azure/autorest/template/ConvenienceMethodTemplateBase.java

```diff
@@ -419,10 +416,8 @@ private static void writeQueryParam(MethodParameter parameter, JavaBlock methodB
             };
         } else {
             writeLine = javaBlock -> javaBlock.line(
-                    String.format("requestOptions.addQueryParam(%1$s, %2$s, %3$s);",
-                            ClassType.String.defaultValueExpression(parameter.getSerializedName()),
-                            expressionConvertToString(parameter.getName(), parameter.getClientMethodParameter().getWireType(), parameter.getProxyMethodParameter()),
-                            parameter.getProxyMethodParameter().getAlreadyEncoded()));
+                    getAddQueryParamExpression(parameter,
```

Comment:
I did not see generated code get updated, but from my understanding, this change will impact generated code.

Reply:
It is in non-azure code branch, so not affect Azure.

Reply:
ok, I see. BTW, may I know why non-azure wants to remove this parameter?

Reply:
I don't know. Maybe they do not want to support the case that parameter already encoded.

But anyway they are not finalized, so they could add it back later. But I had to generate code that compiles with generic-core, righ now...

Reply:
Same for the Trait. I think they will eventually have Trait.

Reply:
I see, thanks.